src/frontend/src/tests/app/AppContext.test.tsx

```diff
@@ -9,7 +9,9 @@ import AppContext from '../../app/AppContext';
 vi.mock('../../app/AppContextQuery', () => {
   return {
     __esModule: true,
-    default: (props: any) => {
+    default: (props: {
+      children: string | number | boolean | {} | React.ReactNodeArray | React.ReactPortal | null | undefined;
```

Comment:
Okay this is interesting! I like that you found a way to make it work, but I do think we should be able to tighten up the props a little bit. Could you try making the options limited to just React.ReactNode or possibly React.ReactNode | null and see if the tests still work?